ui/components/app/currency-input/currency-input.js

```diff
@@ -151,6 +153,7 @@ export default function CurrencyInput({
     if (shouldUseFiat) {
       // Display ETH
       currency = preferredCurrency || EtherDenomination.ETH;
+      suffix = primarySuffix;
```

Comment:
the suffix take only one possible value which is `primarySuffix`, so we dont need to create a mutable variable using `let`, we can simply pass `primarySuffix` as props directly into our component as follows: 
```
return (
      <CurrencyDisplay
        // hides the fiat suffix
        hideLabel={hideSuffix}
        suffix={primarySuffix}  // suggested changes are here
        className="currency-input__conversion-component"
        currency={currency}
        value={newHexValue}
	```

Reply:
Note that `primarySuffix` will be `undefined` when the fiat value is selected. The goal of this – along with `hideLabel={hideSuffix}` – is to only render a suffix for the token, so we don't see things like "$100.37 USD"

Let me know if this makes sense